content/en/docs/concepts/storage/volumes.md

```diff
@@ -1160,6 +1160,33 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```

Comment:
The feature gate page also needs to be updated

Reply:
Done